gcp-out-of-cluster.md

```diff
@@ -17,6 +17,9 @@ GCP users should create [detailed billing export](https://cloud.google.com/billi
 
 ## Step 2:  Visit Kubecost setup page and provide configuration info
 
+
+> **NOTE:** If you are using the alternative [multi-cloud integration](https://docs.kubecost.com/install-and-configure/advanced-configuration/cloud-integration/multi-cloud) method **step 2** is not required.
```

Comment:
Small detail. A comma between "method" and "steps" may increase readability?

Reply:
done